server/handlers/order.js

```diff
@@ -77,3 +78,22 @@ exports.getOrdersByClientID = async function (req, res, next) {
   const orders = result.map((element) => Order.fromMongoJSON(element));
   return res.json(orders);
 };
+
+exports.completeOrderValidatorChain = [orderIDParamValidator];
+
+exports.completeOrderHandler = async function (req, res, next) {
+  // Insert the new order
+  var result;
+  try {
+    result = await dao.completeOrder(req.params.orderID);
```

Comment:
This method doesn't return anything, remove the result variable and also the check for the value.
If the update refers to non existing data, it doesn't update anything and everyone is still happy.